packages/openapi-v3/_spike-request-body.md

```diff
@@ -0,0 +1,110 @@
+## Improve the UX of @requestBody()
+
+The original discussion is tracked in issue
+[Spike: simplify requestBody annotation with schema options](https://github.com/strongloop/loopback-next/issues/2654).
+
+The current @requestBody() can only
+
+- takes in an entire request body specification with very nested media type
+  objects or
+- generate the schema inferred from the parameter type
+
+To simplify the signature, this spike PR introduces two more parameters
+`modelCtor` and `schemaOptions` to configure the schema. The new decorator
+`requestBody2()`(let's discuss a better name later, see section
+[Naming](#Naming) is written in file 'request-body.spike.decorator.ts'
+
+### Signature
+
+The new decorator's signature is
+`@requestBody2(spec, modelCtor, schemaOptions)`.
+
+```ts
+export function requestBody2(
+  specOrModelOrOptions?: Partial<RequestBodyObject> | Function | SchemaOptions,
+  modelOrOptions?: Function | SchemaOptions,
+  schemaOptions?: SchemaOptions,
+) {
+  // implementation
+}
```

Comment:
The reason why I put the private function `_requestBody2()`'s signature(instead of the wrapper's) here is to avoid the confusion caused by manually resolve the caller's input.

Let me change the name to the private one.